src/msgpack/Convertes/ArrayConverter.cs

```diff
@@ -1,7 +1,7 @@
 using System;
 using System.Collections.Generic;
 
-namespace TarantoolDnx.MsgPack
+namespace TarantoolDnx.MsgPack.Convertes
```

Comment:
Please, do not move files around without very good reason for it. VS solution (which @Ivan-Razhnou asked to share) is broken after that. Also, classes in namespace `Converters` should not have `Converter` in name